PowerEditor/src/Parameters.cpp

```diff
@@ -1986,15 +1986,15 @@ bool NppParameters::getSessionFromXmlTree(TiXmlDocument *pSessionDoc, Session *p
 	{
 		if (viewRoots[k])
 		{
-			size_t index2;
+			index = 0;

```

Comment:
See https://github.com/notepad-plus-plus/notepad-plus-plus/commit/d923ece38df175d8f3cccb187786a62bd07de0e3. Might reintroduce an issue.

Reply:
@chcg do you mean reusing `index` variable again or whole PR? 

If you are talking about `index` variable, then I don't see any issue because function ```actIndex->Attribute(TEXT("activeIndex"), &index)``` is going to overwrite something on variable `index` (at least zero in that function). Am I missing something?

If you are talking about whole PR, then I need figure out more about [d923ece](https://github.com/notepad-plus-plus/notepad-plus-plus/commit/d923ece38df175d8f3cccb187786a62bd07de0e3) and what kind of compiling error that PR fixed. But one thing is sure, `size_t` should not be cast to `int` if building for x64. This kind of bugs are obvious while porting 32bit app to 64 bit app which require little extra care.

Reply:
Unfortunately the commit doesn't specify more clearly what the observed bug was, but it introduced the additional variable index2. Maybe it is safer to keep it splitted into two independent variables and be sure to avoid crosstalking between activeIndex and activeView.

Reply:
Okay, done!